resources/src/feedback-form/Project/RatingRadio.js

```diff
@@ -1,19 +1,26 @@
 import React from 'react';
 
-const RatingRadio = ({question_id, radioValue, value, setValue, icon, iconChecked}) => {
-  return (
-    <td className='rating-cell'>
-      <label>
-        <input
-          type='radio'
-          name={question_id}
-          checked={value === radioValue}
-          onChange={() => setValue(radioValue)}
-        />
-        <img src={value === radioValue ? iconChecked : icon} />
+const RatingRadio = ({question_id, radioValue, value, setValue, icon, iconChecked, labelText}) => (
+  <td>
+    <div className={labelText ? 'rating-cell-with-label' : 'rating-cell'}>
```

Comment:
Going by its name, I was really not expecting `labelText` prop to actually be used as a class name, this looks like it's probably a bug?